packages/zilliqa-js-account/src/transaction.ts

```diff
@@ -192,6 +204,60 @@ export class Transaction implements Signable {
     return this;
   }
 
+  /**
+   * blockConfirm
+   *
+   * Use `RPCMethod.GetLatestBlock` to get latest blockNumber
+   * Use interval to get the latestBlockNumber
+   * After BlockNumber change, then we use `RPCMethod.GetTransaction` to get the receipt
+   *
+   * @param {string} txHash
+   * @param {number} maxblockCount
+   * @param {number} interval interval in milliseconds
+   * @returns {Promise<Transaction>}
+   */
+  async blockConfirm(
```

Comment:
@neeboo can you rename this to `confirm` and remove the old confirm? It is not a breaking change, and your technique is much friendlier to the seed node's workload. Thanks!

Reply:
Sure , but different mechanism with different params name, like attempts?

Reply:
I think that's ok. Right now not many third party developers are using this API. So, we can break it. Anyway, we will soon return `Promise<string>` from `createTransaction` soon.